pyproject.toml

```diff
@@ -118,7 +118,8 @@ rich = "^13.7.1"
 psycopg2 = { version = "^2.9.9", optional = true }
 pgvector = { version = "^0.2.5", optional = true }
 structlog = "^24.1.0"
-
+snoop = "^0.4.3"
+litellm = "^1.26.3"
```

Comment:
should these be [optional dependencies](https://github.com/stanfordnlp/dspy/blob/2ad84c2cc9fa34e69402bf775e37e049e90ebcd0/pyproject.toml#L42)?